pkg/apis/openebs.io/v1alpha1/cas_volume.go

```diff
@@ -181,6 +181,8 @@ type CASVolumeSpec struct {
 	TargetPort string `json:"targetPort"`
 	// Replicas will hold the replica count for this volume
 	Replicas string `json:"replicas"`
+	// CasType will hold the engine used to provision this volume
```

Comment:
can you please reword the comment as `storage engine` instead of `engine`

Reply:
done.